kvbc/src/v4blockchain/v4_blockchain.cpp

```diff
@@ -330,6 +330,21 @@ concord::util::digest::BlockDigest KeyValueBlockchain::parentDigest(BlockId bloc
   return block_chain_.getBlockParentDigest(block_id);
 }
 
+concord::util::digest::BlockDigest KeyValueBlockchain::getBlockDigest(BlockId block_id) const {
```

Comment:
nit: I think a better name is calculateBlockDigest because current naming suggests that we can use ```getBlockDigest``` to get  ```parentDigest``` if we pass block_id + 1 , but it's not the case